app/screens/contact/styles.js

```diff
@@ -1,6 +1,6 @@
 import { StyleSheet } from 'react-native';
 
-import colors from 'App/config/colors';
+import colors from '../../config/colors';
```

Comment:
We could create a `package,json` containing the name we want to resolve as in https://medium.com/@davidjwoody/how-to-use-absolute-paths-in-react-native-6b06ae3f65d1 . This was done in Musashi, not sure about Sansaire.

It would resolve something like `'../../../someComponent'` to `'App/someComponent'`. But this is entirely optional.

Reply:
🆗